src/list.js

```diff
@@ -1,38 +1,71 @@
-import React from 'react';
-
+import React from "react";
+import { Link } from "react-router-dom";
 export default class List extends React.Component {
-    constructor(props) {
```

Comment:
why you changing unnecessary file. this is similar to pervious PR, i have mention you already